lib/index.js

```diff
@@ -97,17 +110,27 @@ AdRoll.prototype.track = function(track) {
   del(customProps, 'orderId');
   del(customProps, 'id');
   del(customProps, 'sku');
-  if (!is.empty(customProps)) data.adroll_custom_data = customProps;
+  if (!is.empty(customProps)) {
```

Comment:
haha no idea, i wasn't thinking about refactor existing code at all to keep things *relatively* sane in this change :)